pkg/plugin/vm_backup_item_action.go

```diff
@@ -41,6 +41,12 @@ type VMBackupItemAction struct {
 	log logrus.FieldLogger
 }
 
+const (
+	// AnnMetadataBackup indicates that the object will be backed up for metadata purposes.
+	// This allows skipping restore and consistency-specific checks while ensuring the object is backed up.
+	AnnMetadataBackup = "velero.kubevirt.io/metadataBackup"
```

Comment:
nit: in my opinion the order should be "kubevirt.velero.io" cause this is still in the kubevirt project and after that it related to velero.

Reply:
no we are not "owners" of "velero.io" so "velero.kubevirt.io" is the appropriate order